README.md

```diff
@@ -142,3 +142,19 @@ To disable an openwisp service container and plug your own service like database
    - Change the [email configuration variables](docs/ENV.md) to point to your instances.
 - openwisp-nginx:
    - Configurations in `build/openwisp_nginx/` are helpful to replicate in your own instance.
+
+# Makefile Options
+
+**Right now, this is only tentative guide. Errata may exist. Please report errors on the [gitter channel](https://gitter.im/openwisp/general).**
+
+The Makefile has following options, that are useful for developers & users. Default action is `compose-build`.
+
+`build-base`: Build openwisp-base image. It is the base image used in other images. It takes the most time to build.
+`compose-build`: Build the openwisp images. This option must be used in most usecases, example when a user is building custom images.
+`publish-build`: Similar to `compose-build` except the produced are used for publishing the Official images.
+`runtests`: Run tests on local machine.
+`develop-runtests`: Similar to `runtests` except doesn't stop containers after running the tests.
+`travis-runtests`: Similar to `runtests` except used in travis builds because it doesn't clean up after running the tests and prints all the container's logs when an error occurs in the tests.
+`develop`: Useful for development. Bundles all the commands required to build and run images during development.
```

Comment:
It's not clear to me how this is different from 'publish-build' just from reading the docs.

Reply:
Actually, `travis-runtests` prints the error log of **all** the containers, which is what I don't use when I locally developing because it'll be alot of scrolling before I reach the container with error. Instead, I manually print logs of the container with error which is something I can't do in travis, hence I print the error log of all the containers in `travis-runtests` and not in `develop-runtests`.
Hence, I think logging all the containers isn't very useful in local development.
However, yes, they are very similar, please let me know your thoughts about it. :smile: